app/views/manage/projects/_form.html.haml

```diff
@@ -134,35 +134,31 @@
           %label.form-label
             = form.check_box :group_project
             Group project
-    
-    .row.mb-3#group-manager-row{ style: (@project.group_project ? '' : 'display: none;') }
-      .col-6
-        = form.label :group_manager_id, 'Group Manager', class: 'form-label'
-        .autocomplete#group-manager-autocomplete{ data: { controller: 'autocomplete', autocomplete_url: search_manage_project_finishers_path(@project) } }
-          = text_field_tag :group_manager_name, @project.group_manager&.name, class: 'form-control', data: { autocomplete_target: 'input', action: 'input->autocomplete#fetch' }
-          = form.hidden_field :group_manager_id, value: @project.group_manager_id, data: { autocomplete_target: 'hidden' }
-
-    .row.mb-3
-      .col-12
-        .row
-          %label.form-label
-            = form.check_box :press
-            Press
-    
-    .row.mb-3#press-fields{ style: (@project.press ? '' : 'display: none;') }
-      .col-6
-        = form.label :press_region, 'Region', class: 'form-label'
-        = form.text_field :press_region, class: 'form-control'
-      .col-6
-        = form.label :press_outlet, 'Outlet', class: 'form-label'
-        = form.text_field :press_outlet, class: 'form-control'
-    
-    .row.mb-3
-      .col-12
+        .row.mb-3#group-manager-row{ style: (@project.group_project ? '' : 'display: none;') }
```

Comment:
we would probably want to use the Bootstrap class for this: `hidden` instead of inline styles